### PR TITLE
Improve docker plugin

### PR DESCRIPTION
**Pull request by astj**
Changed graph unit and scaled `blkio.io_serviced` to get IOPS properly.

### CODE REVIEW

**Comment by astj** [comment]
I found `Docker BlkIO IOPS` actually indicates IO per Minute, so now I'm thinking how to handle this... 🤔

File: mackerel-plugin-docker/lib/docker.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ var graphdef = map[string]mp.Graphs{
 	},
 	"docker.memory.#": {
 		Label: "Docker Memory",
-		Unit:  "integer",
+		Unit:  "bytes",
```
Comment (astj) [inline mackerel-plugin-docker/lib/docker.go]: `docker.memory.#.{cache,rss}` are taken from cgroup `memory.stat`

https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt
```
memory.stat file includes following statistics

# per-memory cgroup local status
cache		- # of bytes of page cache memory.
rss		- # of bytes of anonymous and swap cache memory (includes
		transparent hugepages).
```

File: mackerel-plugin-docker/lib/docker.go [hunk 2]
```diff
@@ -55,17 +55,17 @@ var graphdef = map[string]mp.Graphs{
...
+			{Name: "read", Label: "Read", Diff: true, Stacked: true, Type: "uint64", Scale: (1.0 / 60.0)},
+			{Name: "write", Label: "Write", Diff: true, Stacked: true, Type: "uint64", Scale: (1.0 / 60.0)},
+			{Name: "sync", Label: "Sync", Diff: true, Stacked: true, Type: "uint64", Scale: (1.0 / 60.0)},
+			{Name: "async", Label: "Async", Diff: true, Stacked: true, Type: "uint64", Scale: (1.0 / 60.0)},
 		},
 	},
 	"docker.blkio.io_service_bytes.#": {
 		Label: "Docker BlkIO Bytes",
-		Unit:  "integer",
+		Unit:  "bytes",
```
Comment (astj) [inline mackerel-plugin-docker/lib/docker.go]: https://www.kernel.org/doc/Documentation/cgroup-v1/blkio-controller.txt
```
- blkio.io_service_bytes
	- Number of bytes transferred to/from the disk by the group. These
	  are further divided by the type of operation - read or write, sync
	  or async. First two fields specify the major and minor number of the
	  device, third field specifies the operation type and the fourth field
	  specifies the number of bytes.
```

File: mackerel-plugin-docker/lib/docker.go [hunk 3]
```diff
@@ -55,17 +55,17 @@ var graphdef = map[string]mp.Graphs{
...
 	"docker.blkio.io_serviced.#": {
 		Label: "Docker BlkIO IOPS",
-		Unit:  "integer",
+		Unit:  "iops",
 		Metrics: []mp.Metrics{
-			{Name: "read", Label: "Read", Diff: true, Stacked: true, Type: "uint64"},
-			{Name: "write", Label: "Write", Diff: true, Stacked: true, Type: "uint64"},
-			{Name: "sync", Label: "Sync", Diff: true, Stacked: true, Type: "uint64"},
-			{Name: "async", Label: "Async", Diff: true, Stacked: true, Type: "uint64"},
+			{Name: "read", Label: "Read", Diff: true, Stacked: true, Type: "uint64", Scale: (1.0 / 60.0)},
```
Comment (astj) [inline mackerel-plugin-docker/lib/docker.go]: https://www.kernel.org/doc/Documentation/cgroup-v1/blkio-controller.txt
```
- blkio.io_serviced
	- Number of IOs (bio) issued to the disk by the group. These
	  are further divided by the type of operation - read or write, sync
	  or async. First two fields specify the major and minor number of the
	  device, third field specifies the operation type and the fourth field
	  specifies the number of IOs.
```

File: mackerel-plugin-docker/lib/docker.go [hunk 4]
```diff
@@ -55,17 +55,17 @@ var graphdef = map[string]mp.Graphs{
 	},
```
Comment (astj) [inline mackerel-plugin-docker/lib/docker.go]: https://www.kernel.org/doc/Documentation/cgroup-v1/blkio-controller.txt
```
- blkio.io_queued
	- Total number of requests queued up at any given instant for this
	  cgroup. This is further divided by the type of operation - read or
	  write, sync or async.
```

File: mackerel-plugin-docker/lib/docker.go (hunk 1 above)
Comment (astj) [reply mackerel-plugin-docker/lib/docker.go]: btw, in many Linux distributions these stats are always zero (but in Docker for Mac non-zero).
ref: https://docs.docker.com/engine/admin/runmetrics/#memory-metrics-memorystat
```
Therefore, many distros chose to not enable it by default.
```

**Comment by itchyny** [comment]
LGTM

**Comment by astj** [comment]
Thanks.

**Pull request merged**